sceptre/scipool/config/bmgfki/sc-product-s3-private-enc.yaml

```diff
@@ -55,7 +55,11 @@ sceptre_user_data:
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.33/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.33'
-    - Description:  'Disable bucket ACLs. {{ range(1, 10000) | random }}'
+    - Description:  'Disable bucket ACLs.'
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.38/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.38'
+    - Description:  'Update cfn-cr-same-region-bucket-download lambda. {{ range(1, 10000) | random }}'
```

Comment:
this description seems too generic.  could we be more specific on what changed?

Reply:
I've changed it to specify that we're updating the python version used.